hugegraph-mysql/src/main/java/com/baidu/hugegraph/backend/store/mysql/MysqlSessions.java

```diff
@@ -222,19 +222,25 @@ protected Connection openWithoutDB(int timeout) {
         private Map<String, PreparedStatement> statements;
         private boolean opened;
         private int count;
+        private HugeConfig config;
 
         public Session() {
             this.conn = null;
             this.statements = new HashMap<>();
             this.opened = false;
             this.count = 0;
+            this.config = MysqlSessions.this.config();
```

Comment:
unneeded to define this.config, it's ok to return `MysqlSessions.this.config()` in config()